client/services/code_preprocessors/PythonCodePreprocessorService.js

```diff
@@ -165,9 +165,13 @@ tie.factory('PythonCodePreprocessorService', [
         VARNAME_CORRECTNESS_TEST_RESULTS + ' = []',
       ].join('\n');
       for (var i = 0; i < correctnessTests.length; i++) {
+        var testInputCode = (
+          inputFunctionName ? inputFunctionName + '(' + VARNAME_TEST_INPUTS + 
```

Comment:
I think it would be cleaner if you break after the '?' and include parentheses, i.e.

    var testInputCode = (
      inputFunctionName ?
      (bunch of complicated stuff here) : 
      VARNAME_TEST_INPUTS + '[' + i + ']'
    );